sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -1,13 +1,17 @@
 # Release History
-## 1.2.0-beta.1 (Unreleased)
-
-### Features Added
+## 2.0.0 (2022-08-26)
+### Bugs Fixed
+- Push notification registration renewal not guaranteed to happen when Contoso APP is idle or closed.
```

Comment:
Are we saying this is no longer happening? The wording is confusing and could make people think the renewal is not guaranteed to happen when in fact, it is.
```suggestion
- Push notification registration renewal is now guaranteed to happen when an app is idle or closed.
```